Italian/smartphone/smartphone.md

```diff
@@ -3,6 +3,8 @@ Ma 
 Cerco disperatamente di accedere al GPS ma non ottengo nessuna risposta.
 Allora vince il riflesso condizionato e penso a quale social mi potrebbe essere pi� utile nell'occasione.
 
-Scelgo [Facebook](facebook/facebook.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.